flocker/node/functional/test_docker.py

```diff
@@ -67,17 +70,13 @@ class GenericDockerClientTests(TestCase):
     Functional tests for ``DockerClient`` and other clients that talk to
     real Docker.
     """
+    clientException = APIError
```

Comment:
I was about to say something about this variable name, but I see it's always been that way.